graphql/resolvers.go

```diff
@@ -1133,8 +1133,15 @@ func (r *mutationResolver) UpdateUserSettings(ctx context.Context, userSettings
 	return true, nil
 }
 
-func (r *queryResolver) User(ctx context.Context) (*restModel.APIUser, error) {
+func (r *queryResolver) User(ctx context.Context, userIdParam *string) (*restModel.APIUser, error) {
 	usr := route.MustHaveUser(ctx)
+	var err error
+	if userIdParam != nil {
+		usr, err = model.FindUserByID(*userIdParam)
+		if err != nil {
+			return nil, InternalServerError.Send(ctx, fmt.Sprintf("Error getting user from user ID: %s", err.Error()))
+		}
```

Comment:
what if user is equal to nil? should we check for that and send a ResourceNotFound?

Reply:
is user is nil, err will not be nil. check it out here: https://github.com/evergreen-ci/evergreen/blob/master/model/user_lifecycle.go#L30